cli/dcoscli/tables.py

```diff
@@ -70,6 +70,23 @@ def get_container(app):
         else:
             return "mesos"
 
+    def is_overdue(row):
+        """Calculates if given app is overdue
```

Comment:
You can define a `backoff` duration in seconds in the app definition. If the starting of this application will take longer than this duration, the application will be marked as overdue.

Will add a comment.